docs/src/components/copy-command-button.tsx

```diff
@@ -0,0 +1,68 @@
+'use client';
+
+import { useState } from 'react';
+import { copyTextToClipboard } from '../utils';
+import Button from './button';
+
+export function CopyCommandButton() {
+  const copyCommandText = 'npm create @keystatic@latest';
+  const [isCopied, setIsCopied] = useState(false);
+
+  const handleCopyCommand = () => {
+    copyTextToClipboard(copyCommandText)
+      .then(() => {
+        setIsCopied(true);
+        setTimeout(() => {
+          setIsCopied(false);
+        }, 1500);
+      })
+      .catch(err => {
+        console.log(err);
+      });
+  };
+
+  return (
+    <Button
+      className="border-keystatic-gray items-center py-0"
```

Comment:
On my screen the CLI command box looks a bit washed out and gets lost in the rest of the page:

<img width="661" alt="CleanShot 2023-06-20 at 12 33 08@2x" src="https://github.com/Thinkmill/keystatic/assets/5933897/758c39db-e01b-4699-9fda-e765c9a94ef3">

Do you think we can make it stand out a bit more? Different background or darker border colour?

Basically I'm asking if you can make it pop. 😄

Reply:
Haha yeah I'll just make it the normal secondary Button style then and it'll pop more!